README.md

```diff
@@ -7,26 +7,31 @@ Docker powered mini-Heroku. The smallest PaaS implementation you've ever seen.
 
 ## Requirements
 
-- A fresh VM running `Debian 8.2 x64`, `Ubuntu 14.04 x64`, or `Ubuntu 16.04 x64`
+A fresh VM running any one of 
+- `Debian 8.2 x64`
+- `Ubuntu 14.04 x64`
+- `Ubuntu 16.04 x64`
 
-## Installing
+## Installation
 
-To install the latest stable release, you can run the following commands as a user that has access to `sudo`:
+To install the latest stable release, run the following commands as a user who has access to `sudo`:
 
-    wget https://raw.githubusercontent.com/dokku/dokku/v0.7.2/bootstrap.sh
-    sudo DOKKU_TAG=v0.7.2 bash bootstrap.sh
+```
```

Comment:
This doesn't bring any benefit if you aren't going to add `shell` as a highlight.